src/traces/indicator/defaults.js

```diff
@@ -35,6 +35,10 @@ function supplyDefaults(traceIn, traceOut, defaultColor, layout) {
         coerce('number.valueformat');
         coerce('number.font.color', layout.font.color);
         coerce('number.font.family', layout.font.family);
+        coerce('number.font.weight', layout.font.weight);
+        coerce('number.font.style', layout.font.style);
+        coerce('number.font.stretch', layout.font.stretch);
+        coerce('number.font.variant', layout.font.variant);
```

Comment:
Is there a reason indicator doesn’t use `coerceFont`?

Reply:
Addressed by 2d7ba1d.